termsaverlib/common.py

```diff
@@ -233,7 +233,7 @@ def get_mem_usage(ignore_errors=False):
             
         elif is_macos():
 
-            vm = subprocess.Popen(['vm_stat'], stdout=subprocess.PIPE).communicate()[0].decode()
+            vm = subprocess.Popen(['vm_stat'], stdout=subprocess.PIPE).communicate()[0] #.decode()
```

Comment:
Have you tested this on a MacOS machine?

Reply:
I just tested this on a fresh Catalina install and the only module that gave any error was sysmon. If you'd like, I can investigate further at a future point and make it an issue specific to sysmon, but as I said, all testable modules work fine otherwise.

Reply:
I don't remember anymore why I had the decode there, but looks in python3 this might be resolved. If you don't have a Mac (I don't anymore), and you cannot find anyone with one for us to test, then just leave it.

Reply:
> 
> 
> I don't remember anymore why I had the decode there, but looks in python3 this might be resolved. If you don't have a Mac (I don't anymore), and you cannot find anyone with one for us to test, then just leave it.

I do have a mac! It's just a machine primarily for novels and research, not development, but getting Python3 wasn't a huge hassle so I can easily use it for testing.